### PR TITLE
Hotfix PageLayout, DotIcon, Sveltekit init

### PR DESCRIPTION
**Pull request by hypervillain**
Completes https://linear.app/prismic/issue/DT-1778/aapm-i-can-track-when-a-user-has-switch-environment, Completes https://linear.app/prismic/issue/DT-1790/aadev-i-want-to-comment-text-regular-environments-in-envsswitch, Completes 
 https://linear.app/prismic/issue/DT-1789/aadev-i-want-to-be-able-to-write-an-importmetaenv-string-without-it

### CODE REVIEW

File: packages/slice-machine/src/components/PageLayout/PageLayout.stories.tsx [hunk 1]
```diff
@@ -16,9 +16,9 @@ const meta = {
     children: {
       control: { disable: true },
     },
-    activeEnvironmentKind: {
+    environmentKind: {
       control: { type: "select" },
-      options: Object.keys(borderTop),
+      options: [Object.keys(borderTop), undefined],
```
Comment (xrutayisire) [inline packages/slice-machine/src/components/PageLayout/PageLayout.stories.tsx]: ⚠️ You need to spread here right?

File: packages/adapter-sveltekit/src/hooks/project-init.ts [hunk 2]
```diff
@@ -62,8 +62,7 @@ const createPrismicIOFile = async ({
 			 * The project's Prismic repository name.
 			 */
 			export const repositoryName =
-				import.meta.env.${PRISMIC_ENVIRONMENT_ENVIRONMENT_VARIABLE_NAME} ||
-				config.repositoryName;
+				import${"."}meta${"."}env.${PRISMIC_ENVIRONMENT_ENVIRONMENT_VARIABLE_NAME} || config.repositoryName;
```
Comment (xrutayisire) [inline packages/adapter-sveltekit/src/hooks/project-init.ts:65]: ⚠️ We need handle this for js right?

Same for Next?!

I guess we have another bug in prod right?

File: packages/slice-machine/src/components/SideNav/SideNavEnvironmentSelector.tsx [hunk 3]
```diff
@@ -94,7 +94,10 @@ const EnvironmentDropdownMenu: FC<EnvironmentDropdownMenuProps> = (props) => {
         <IconButton icon="unfoldMore" />
       </DropdownMenuTrigger>
       <DropdownMenuContent align="end" minWidth={256}>
-        <DropdownMenuLabel>Regular Environments</DropdownMenuLabel>
+        {/*
```
Comment (xrutayisire) [inline packages/slice-machine/src/components/SideNav/SideNavEnvironmentSelector.tsx:97]: Nice!

File: packages/slice-machine/src/components/PageLayout/PageLayout.tsx [hunk 4]
```diff
@@ -3,17 +3,21 @@ import type { FC, PropsWithChildren } from "react";
 import * as styles from "./PageLayout.css";
 
 type PageLayoutProps = PropsWithChildren<{
-  activeEnvironmentKind?: "prod" | "stage" | "dev";
+  environmentKind?: "prod" | "stage" | "dev";
```
Comment (xrutayisire) [inline packages/slice-machine/src/components/PageLayout/PageLayout.tsx]: ℹ️ Just so you know, PageLayout should be product functional agnostic, meaning no mention to environement here. @bapmrl will open a PR I think for the future.

**Review by xrutayisire — CHANGES_REQUESTED** [review]
I think I found another bug for JS 😛

File: packages/slice-machine/src/components/PageLayout/PageLayout.tsx (hunk 4 above)
Comment (hypervillain) [reply packages/slice-machine/src/components/PageLayout/PageLayout.tsx]: Yep makes sense

**Comment by hypervillain** [comment]
@bapmrl you can find the PageLayout fix here too

**Review by xrutayisire — APPROVED** [review]
Nice fixes!

**Pull request merged**